index.d.ts

```diff
@@ -39,6 +39,14 @@ declare namespace globby {
 		@default false
 		*/
 		readonly gitignore?: boolean;
+
+		/**
+		Glob pattern to look for ignore files, which are then used to ignore globbed files.
+		This value is unused if gitignore option is used, since gitignore takes precedence.
```

Comment:
This is ambiguous, setting `gitignore: false` is technically *it being used*. It should explicitly say that setting `gitignore: true` overrides this option.

Reply:
Done.